all-imports.html

```diff
@@ -1,3 +1,3 @@
 <!-- all-imports.html -->
 <link rel="import" href="i18n-msg.html">
-<link rel="import" href="i18n-page-example.html">
+<link rel="import" href="demo/i18n-msg-behavior-example.html">
```

Comment:
[Following this documentation "By default, iron-component-page assumes a single import, with a name matching the folder name."](https://www.polymer-project.org/1.0/tools/documentation.html)

I have added i18n-msg-behavior-example in the documentation as the i18-msg. You think i18n-msg-behavior-example should not be in the documentation ? The behavior is automatically added like you see in the following screen.

![i18nmsg](http://img4.hostingpics.net/pics/803600i18nmsg.jpg)